scripts/Muon/GUI/Common/utilities/load_utils.py

```diff
@@ -218,35 +218,50 @@ def load_workspace_from_filename(filename,
         load_result = _get_algorithm_properties(alg, output_properties)
         load_result["OutputWorkspace"] = [MuonWorkspaceWrapper(ws) for ws in load_result["OutputWorkspace"]]
         run = get_run_from_multi_period_data(workspace)
-        load_result["DataDeadTimeTable"] = copy.copy(load_result["DeadTimeTable"][0])
-        load_result["FirstGoodData"] = round(load_result["FirstGoodData"] - load_result['TimeZero'], 2)
+        if not is_psi_data:
```

Comment:
can we have `if psi_data` as its just nicer to read

Reply:
Done!